spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe 'likes' do
+
+  context 'a post is added and a non signed-in user is viewing that post' do
+
+    let! (:user) { create(:user) }
+    let! (:post) { user.posts.create(FactoryGirl.attributes_for(:post)) }
+
+    before do
+      visit '/'
+    end
+
+    scenario 'user is promted with a Like button' do
+      expect(page).to have_content 'Like'
+      expect(page).not_to have_button 'Unlike'
+    end
+
+    scenario 'user trying to like a post is asked to log in' do
+      click_on 'Like'
+      expect(page).to have_content 'You need to sign in or sign up before continuing.'
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)